person_class.rb

```diff
@@ -8,15 +12,27 @@ def initialize(age:, name: 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super()
   end
 
   def can_use_services?
     is_of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def is_of_age? # rubocop:disable Naming/PredicateName
     @age >= 18
   end
 end
+
+person = Person.new(age: 22, name: 'maximilianus')
+person.correct_name
+capitalizedperson = CapitalizeDecorator.new(person)
+capitalizedperson.correct_name
+capitalizedtrimmedperson = TrimmerDecorator.new(capitalizedPerson)
+capitalizedtrimmedperson.correct_name
```

Comment:
- [ ] A `NameError` occurs here because you have a typo in the `capitalizedPerson` variable. I believe you meant to write `capitalizedperson` instead 👍 
![image](https://user-images.githubusercontent.com/46839548/223728990-073f80c1-82c7-443d-b604-086f042fefe5.png)